ui/src/views/Tasks/CreateTask/index.jsx

```diff
@@ -55,14 +52,78 @@ const defaultTask = {
     ],
     // 30s margin to avoid task timeout winning race against task command.
     maxRunTime: 600 + 30,
-  },
+  };
+
+  if (interactive) {
+    defaultPayload = {
+      ...defaultPayload,
+      maxRunTime: 3600,
+      features: {
+        interactive: true,
+      },
+    };
+  }
+
+  return defaultPayload;
+};
+
+const defaultTask = interactive => ({
+  provisionerId: 'aws-provisioner-v1',
+  workerType: 'tutorial',
+  created: new Date().toISOString(),
+  deadline: toDate(addHours(new Date(), 3)).toISOString(),
+  payload: taskPayload(interactive),
   metadata: {
     name: 'Example Task',
     description: 'Markdown description of **what** this task does',
     owner: 'name@example.com',
     source: `${window.location.origin}/tasks/create`,
   },
-};
+});
+const DialogInteractiveChangeText = () => (
+  <Fragment>
+    Selecting an interactive task will apply the following changes to the task
+    definition:
+    <List>
+      <ListItem>
+        <ListItemText>
+          Set
+          <span className="CodeMirror">
+            task.payload.features.interactive = true
+          </span>
+        </ListItemText>
+      </ListItem>
+      <ListItem>
+        <ListItemText>
+          Strip
+          <span className="CodeMirror">task.payload.caches</span>
+          to avoid poisoning
+        </ListItemText>
+      </ListItem>
+      <ListItem>
+        <ListItemText>
+          Ensures
+          <span className="CodeMirror">task.payload.maxRunTime</span>
+          is minimum of 60 minutes
+        </ListItemText>
+      </ListItem>
+      <ListItem>
+        <ListItemText>
+          Strip
+          <span className="CodeMirror">task.routes</span> to avoid side-effects
+        </ListItemText>
+      </ListItem>
+      <ListItem>
+        <ListItemText>
+          Set the environement variable
+          <span className="CodeMirror">TASKCLUSTER_INTERACTIVE=true</span>
```

Comment:
I think `<code>` can replace `<span className="CodeMirror">`.

Reply:
Many of the list items are missing a space before the inline code.

![Screen Shot 2019-10-29 at 10 47 03 AM](https://user-images.githubusercontent.com/3766511/67778378-7b496c00-fa39-11e9-8641-1570ef3662f6.png)